examples/notification_ui_integration.py

```
@@ -0,0 +1,301 @@
+"""
+Example script demonstrating how to integrate the notification system with a UI.
+
+This script shows how to create a simple notification center using the notification system.
+"""
+
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)